docs/src/development-guide/building.md

```diff
@@ -29,11 +41,38 @@ In order to run this app on iOS you have to open the Xcode project at `./apple/x
 You can then run the app on an iOS Simulator or a real device. During the Xcode build process cargo is used to build
 a static library for the required architecture.
 
-## Desktop (macOS)
+## MacOS
 
-In order to run this app on macOS you have to open the Xcode project at `./apple/xcode`.
-You can then run the app on a macOS. During the Xcode build process cargo is used to build
-a static library for the required architecture.
+You can build Unix Executables for MacOS, as explained in the first section, with one of the following toolchain targets:
+
+* **x86_64-apple-darwin** for Intel's x86-64 processors
+* **aarch64-apple-darwin** for ARM64 processors
+
+Use one of the following commands to build for your desired target architecture:
+
+```bash
+cargo build -p maplibre-demo --target x86_64-apple-darwin
+cargo build -p maplibre-demo --target aarch64-apple-darwin
+```
+
+If you want to build a proper MacOS application (in OSX terminology), you will need to use the XCode project
+in the folder `./apple/xcode/`.
+
+After installing [XCode](https://apps.apple.com/us/app/xcode/id497799835?ls=1&mt=12) and [Rustup](https://rustup.rs/),
+build the Rust library inside `./apple` with Cargo:
+
+```bash
+cargo build -p apple --lib
+```
```

Comment:
This is no longer needed, but maybe we should point out that the XCode project is executing tomething like that using the correct `--target` flag.

Reply:
I have removed this part and I have added a little text to mention that :

> The XCode project is configured to automatically compile the Rust library with the correct target in the *Cargo Build* build phases configuration.